### PR TITLE
Make 'Import Tracks' working with single clip

### PR DESCRIPTION
**Pull request by bartekd97**
I've found out that 'Import Tracks' options while importing AnimationPlayer was not working at all if there was only a one animation clip, so you couldn't e.g. omit 'Scale' tracks.
Fixed it.

image

### CODE REVIEW

**Comment by akien-mga** [comment]
Sorry for the delay. This looks fine to me, though I'm not an expert. Would be good if @godotengine/animation @godotengine/import folks could check. Also needs a rebase.

File: editor/import/resource_importer_scene.cpp [hunk 1]
```diff
@@ -1030,7 +1030,16 @@ Node *ResourceImporterScene::_post_fix_node(Node *p_node, Node *p_root, Map<Ref<
 				AnimationImportTracks(int(node_settings["import_tracks/scale"]))
 			};
 
-			if (anims.size() > 1 && (import_tracks_mode[0] != ANIMATION_IMPORT_TRACKS_IF_PRESENT || import_tracks_mode[1] != ANIMATION_IMPORT_TRACKS_IF_PRESENT || import_tracks_mode[2] != ANIMATION_IMPORT_TRACKS_IF_PRESENT)) {
+			// optimize track usage only if import options differ from default behaviour
```
Comment (akien-mga) [inline editor/import/resource_importer_scene.cpp:1033]:
```suggestion
			// Optimize track usage only if import options differ from default behavior.
```

**Comment by fire** [comment]
Needs a rebase.

There are some conflicts that arise when 3D animation tracks are not present, causing problems in the project. The exact issue can vary depending on the specific situation, but generally, these problems occur due to missing or incomplete data for the animations. So need to check when omitting tracks if it causes problems.

**Comment by lyuma** [comment]
Superseded by #78495
it really was `size() > 1` which should be `size() >= 1` or `!is_empty()` which is what I did.

**Pull request closed without merging**